src/sass/components/_contact-form.scss

```diff
@@ -1,10 +1,18 @@
 .contacts-form__input {
-  width: 100%;
   border: none;
   border-bottom: 1px solid var(--accentColor);
   display: block;
   background-color: transparent;
 
+  width: 265px;
+
+  @media screen and (min-width: 767px) {
```

Comment:
767px > $tablet

Reply:
@media screen and (min-width: 768px) {
    width: 410px;